interfaces/builtin/polkit_test.go

```diff
@@ -109,9 +109,10 @@ func (s *polkitInterfaceSuite) TestConnectedPlugPolkit(c *C) {
 </policyconfig>`
 	const samplePolicy2 = `<policyconfig/>`
 
-	policyPath := filepath.Join(s.plugInfo.Snap.MountDir(), "meta/polkit.foo.policy")
+	c.Assert(os.MkdirAll(filepath.Join(s.plugInfo.Snap.MountDir(), "meta/polkit"), 0755), IsNil)
+	policyPath := filepath.Join(s.plugInfo.Snap.MountDir(), "meta/polkit/polkit.foo.policy")
```

Comment:
I suppose these example could drop the "polkit." part of the filenames?  meta/polkit/foo.policy ?